package.json

```diff
@@ -1,24 +1,30 @@
 {
-    "name": "eslint-config-scality",
+    "name": "@scality/eslint-config",
     "version": "7.10.2",
     "description": "ESLint config for Scality's Node.js coding guidelines",
     "bin": {
-        "mdlint": "./bin/mdlint.js"
+        "mdlint": "bin/mdlint.js"
     },
     "main": "index.js",
     "scripts": {
         "test": "npm run --silent lint && npm run --silent lint_md",
-        "lint": "node_modules/.bin/eslint -c index.js $(git ls-files '*.js')",
-        "lint_md": "node bin/mdlint.js $(git ls-files '*.md')"
+        "lint": "eslint -c index.js $(git ls-files '*.js')",
+        "lint_md": "node bin/mdlint.js $(git ls-files '*.md')",
+        "get-version": "echo $npm_package_version",
```

Comment:
maybe simpler to just run `jq .version < package.json` / `jq .name < package.json` / `basename $(jq .name < package.json)` instead of `yarn get-version`/... and extra npm scripts....

Reply:
done